packages/ui/src/components/va-data-table/VaDataTable.vue

```diff
@@ -429,6 +430,7 @@ export default defineComponent({
 
   &--sticky {
     overflow-y: auto;
+    height: var(--sticky-table-height);
```

Comment:
Can't we set height if the table is not sticky? 🤔

Reply:
Added